src/Pim/Bundle/CatalogBundle/Resources/config/query_builders.yml

```diff
@@ -67,11 +68,9 @@ services:
 
     # PQB common filter
     pim_catalog.doctrine.query.filter.category:
-        class: '%pim_catalog.doctrine.query.filter.category.class%'
+        class: '%pim_catalog.elasticsearch.query.filter.category.class%'
         arguments:
             - '@pim_catalog.repository.category'
```

Comment:
the service exists
how can you conclude the service does not exist because PHPstorm does not find it?

burn your mac maybe
or use `nano` :trollface: